datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/resolvers/ingest/execution/CreateTestConnectionRequestResolver.java

```diff
@@ -74,7 +74,6 @@ public CompletableFuture<String> get(final DataFetchingEnvironment environment)
 
         Map<String, String> arguments = new HashMap<>();
         arguments.put(RECIPE_ARG_NAME, input.getRecipe());
-        arguments.put(VERSION_ARG_NAME, _ingestionConfiguration.getDefaultCliVersion());
```

Comment:
This will use the latest?

Reply:
correct - Shirshanka just pushed a change to use latest if we don't supply a cli version